azure_arc_servers_jumpstart/docs/arc_updateManagement.md

```diff
@@ -0,0 +1,190 @@
+
+    ![](../img/updateManagement/01.png)
+
+    ![](../img/updateManagement/02.png)
+
+  
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.14** or later. Use ```az --version``` to check your current installed version.
+
+* Create Azure Service Principal (SP).   
+
+    To connect a VM or bare-metal server to Azure Arc, Azure Service Principal assigned with the "Contributor" role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```bash
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+    For example:
+    ```az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor```
+    Output should look like this:
+    ```
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+    
+  **Note**: It is optional but highly recommended to scope the SP to a specific [Azure subscription and Resource Group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).
+
+# Configuring Update Management
+
+Update Management uses the Log Analytics agent to collect Windows and Linux server log files and the data collected is stored in a Log Analytics workspace. 
+
+* You will need to create a Log Analytics workspace. For that you can use this [ARM template](../updateManagement/law-template.json) that will create a new Log Analytics Workspace and define the Azure Sentinel solution and enable it for the workspace. 
+
+* First, create a new resource group for the Log Analytics workspace by running the below command, replacing the values in brackets with your own.
+
+    ```bash
+    az group create --name <Name for your resource group> --location <Location for your resources>
+    ```
+    ![](../img/updateManagement/03.png)
+
+* Next, edit the ARM template [parameters file](../updateManagement/law-template.parameters.json), providing a name for your Log Analytics workspace, a location, and a name for your Azure Automation account as shown in the example below:
+
+    ![](../img/updateManagement/05.png)
+
+* To deploy the ARM template, navigate to the [deployment folder](../updateManagement/) and run the below command:
+
+    ```bash
+    az deployment group create --resource-group <Name of your Azure Resource Group> \
+        --template-file law-template.json \
+        --parameters law-template.parameters.json
+    ```
+
+   ![](../img/updateManagement/04.png)
```

Comment:
Switch 04/05 png's

Reply:
these are just numbered out of order, the screens themselves are in the right order. If you insist, I'll rename all the screenshots so the numbers line up but as they are now the screens themselves are in the right order.